aiohttp/client_reqrep.py

```diff
@@ -519,10 +519,12 @@ async def write_bytes(
                 await self.body.write(writer)
             else:
                 if isinstance(self.body, (bytes, bytearray)):
-                    self.body = (self.body,)  # type: ignore
+                    body = (self.body,)
+                else:
+                    body = self.body
```

Comment:
This looks like a logic change unrelated to typing

Reply:
I've restored the logic